libs/membership/editor/src/lib/subscription-flow/timeline/timeline-body.tsx

```diff
@@ -46,7 +46,7 @@ export function TimelineBody({days, subscriptionFlow, eventIcons, eventColors}:
   }
 
   return (
-    <>
```

Comment:
This fragment seems correct here and looks more like eslint unable to understand we return multiple here

Reply:
yeah I know :D but I would still keep it as it is